docs/index.md

```diff
@@ -1,12 +1,5 @@
-# ScalarDB
+# Add ScalarDB to your build
```

Comment:
Can you remove lines after lines 22 since they are unrelated to adding ScalarDB to your build?

Reply:
Updated. PTAL!